internal/quantity/quantity.go

```diff
@@ -36,19 +36,53 @@ func MulFloat64(x resource.Quantity, y float64) (resource.Quantity, error) {
 	return Mul(x, yQuantity), nil
 }
 
-func Div(x resource.Quantity, y resource.Quantity) resource.Quantity {
+func Div(x resource.Quantity, y resource.Quantity, s inf.Scale) resource.Quantity {
 	result := resource.Quantity{}
 	result.Format = x.Format
 
-	result.AsDec().QuoRound(x.AsDec(), y.AsDec(), 0, inf.RoundUp)
+	result.AsDec().QuoRound(x.AsDec(), y.AsDec(), s, inf.RoundUp)
 	return result
 }
 
-func DivFloat64(x resource.Quantity, y float64) (resource.Quantity, error) {
+func DivFloat64(x resource.Quantity, y float64, s inf.Scale) (resource.Quantity, error) {
 	yQuantity, err := resource.ParseQuantity(fmt.Sprintf("%v", y))
 	if err != nil {
 		return x, err
 	}
 
-	return Div(x, yQuantity), nil
+	return Div(x, yQuantity, s), nil
+}
+
+func Min(x resource.Quantity, y resource.Quantity) resource.Quantity {
+	xCopy := x.DeepCopy()
+	yCopy := y.DeepCopy()
+
+	if xCopy.Cmp(yCopy) == -1 {
+		return xCopy
+	}
+
+	return yCopy
+}
+
+func Max(x resource.Quantity, y resource.Quantity) resource.Quantity {
+	xCopy := x.DeepCopy()
+	yCopy := y.DeepCopy()
+
+	if xCopy.Cmp(yCopy) == -1 {
+		return yCopy
+	}
+
+	return xCopy
+}
+
+// convenience function for testing only
+func Ptr(q resource.Quantity) *resource.Quantity {
+	return &q
+}
```

Comment:
This should be moved into a `_testing` helper file if it is not meant to be exported as part of the package.

Reply:
My bad, this can be included as part of the package, removed the comment.

Reply:
Why is this function needed?  Couldn't a caller of the function simply replace it with `&q`